services/app-web/src/common-code/dateHelpers/calendarDate.ts

```diff
@@ -1,12 +1,21 @@
 import { dayjs } from './dayjs'
 
-// We store calendar dates (dates with no time info) in UTC for consistency.
-// This formats a date correctly like '12/02/2022' for display.
-function formatCalendarDate(date: Date | undefined): string {
-    if (!date) {
+// We store calendar dates in UTC for consistency. This formats a date like '12/02/2022'.
+// These formatted dates, should only be used for display, never saved to the database.
+function formatCalendarDate(date: Date | undefined | string, timeZone?: 'UTC' | 'America/New_York'): string {
+    const tz = timeZone || 'UTC'
```

Comment:
I would like to see this be a little more developer foolproof.  I'm worried someone will forget to pass a timezone and then we get the same problem.

Let's make the timeZone prop required. You must defined what you are doing to display the date. And then add a comment that **user actions are always displayed in eastern time**, form data like start and end dates of contracts are displayed in UTC.